tools/ingest/retrieve_data_from_noaa_nomads.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
```

Comment:
Suggestion: `/bin/bash`

You reference the use of `bash` on line 19 in the _usage_ statement setting.

Also: set the exec bits for the script and obviate the need for exec'ing a (sub) shell when running the script.

Reply:
will change line 1

try to always set scripts with the exec bits on initial commit will make sure they are set, but still usually from old habits launch as a subshell when running scripts

will leave the recommended _usage_ statement unless large objection

Reply:
If you set the script to be executable, then it can be launched as

`script args`

and not as

`bash script args` (which starts a sub-shell)

This would necessarily change your `usage` statement.

These are only suggestions; I'm not a reviewer.